src/components/IconAlertSuccess.vue

```diff
@@ -0,0 +1,40 @@
+<template>
```

Comment:
Kan vi ta i bruk den fra [Punkt](https://punkt.oslo.kommune.no/latest/profil/ikoner/index) her? Eller er det en mye større jobb?

Reply:
Det synes jeg vi skal; umiddelbart litt usikker på hvordan vi best gjør det, for å bytte alle, så lurer på om det kan være greit å se på det separat 🤔